.palantir/revapi.yml

```diff
@@ -249,3 +249,8 @@ acceptedBreaks:
       new: "method <T> T com.palantir.dialogue.Clients::callBlocking(com.palantir.dialogue.EndpointChannel,\
         \ com.palantir.dialogue.Request, com.palantir.dialogue.Deserializer<T>)"
       justification: "New API for blocking calls."
+  "2.10.0":
+    com.palantir.dialogue:dialogue-target:
```

Comment:
Assuming this version needs to be bumped to the release version?

Reply:
It’s a bit of a race, the version isn’t revalidated and that won’t break anything.